### PR TITLE
Removed net standard 2.0

### PR DESCRIPTION
**Pull request by ReeceHumphreys**
Removed the build folders related to net standard 2.0
- Followed the instructions from the csharp readme for macOS and everything else still built fine.
- Not sure if I did indeed remove everything properly as the csharp build system is confusing, so let me know if something doesn't make sense.

### CODE REVIEW

**Review by pepone — APPROVED** [review]
Looks good!

**Comment by pepone** [comment]
The main README need updates remove the corresponding section https://github.com/ReeceHumphreys/ice-demos/tree/removing-netstandard/csharp#building-and-running-the-demos-with-net-core-on-windows-linux-and-macos

File: csharp/C# NET Core demos.sln [hunk 1]
```diff
@@ -1,1253 +0,0 @@
-Microsoft Visual Studio Solution File, Format Version 12.00
```
Comment (externl) [inline csharp/C# NET Core demos.sln:1]: Is this the right file to delete?

Comment (pepone) [reply csharp/C# NET Core demos.sln:1]: Yes that is fine

**Review by externl — APPROVED** [review]
Looks good!

**Pull request merged**